release_notes_generator/record/record_factory.py

```diff
@@ -0,0 +1,88 @@
+import logging
+
+from github import Github
+from github.Issue import Issue
+from github.PullRequest import PullRequest
+from github.Repository import Repository
+from github.Commit import Commit
+
+from release_notes_generator.model.record import Record
+
+from release_notes_generator.utils.decorators import safe_call_decorator
+from release_notes_generator.utils.github_rate_limiter import GithubRateLimiter
+from release_notes_generator.utils.pull_reuqest_utils import extract_issue_numbers_from_body
+
+
+class RecordFactory:
```

Comment:
Again, only static method in a class does not need class.

Reply:
What is your fix opinion here? Remove class and provide only method?